src/support/mod.rs

```diff
@@ -1,20 +1,23 @@
-mod arbitrary;
-mod ark_ff;
-mod bn_rs;
-mod fastrlp;
-mod num_bigint;
-mod postgres;
-mod primitive_types;
-mod proptest;
-mod pyo3;
-mod quickcheck;
-mod rand;
-mod rlp;
-mod scale;
-mod serde;
-mod sqlx;
-mod valuable;
-mod zeroize;
+//! Support for external crates.
+
+pub mod alloy_rlp;
```

Comment:
do any of these modules contain exported types?

Reply:
Some of them, should I make the empty ones private?

Reply:
looks like only sqlx & scale have exported items. lets keep the other ones private to avoid cluttering user namespace